docs/API/InstrumentFormat.md

```diff
@@ -90,9 +120,13 @@ the following format:
     "Type" : string,
     "Name" : UniqueQuestionIdentifier,
     "Description" : "Human readable question text",
+    "DisplayIf" : string, /* LORIS Logic Parser formula */
+    "HiddenSurvey" : string, /* LORIS Logic Parser formula */
+    "Hidden" : string, /* LORIS Logic Parser formula */
```

Comment:
I can't really review this independently of the "LORIS Logic Parser formula" spec which isn't included. (Sorry, I know I told you to send them separately, but I didn't realize this would be dependent on it.)

But given the paragraph up on line 21, I think this should be documented in a separate rules document and define the rules separately from the instrument layout so that they can be evolved independently. (It would be easier to do that if these were moved to a separate key on the object, maybe "DisplayRules" or something of the sort.

Reply:
I'll create a PR for that doc today.